.travis.yml

```diff
@@ -18,12 +10,37 @@ addons:
       - ldap-utils
       - slapd
 
+matrix:
+  include:
+    - python: 2.7
+      env:
+        - TOXENV=py27
+        - WITH_GCOV=1
```

Comment:
Is `WITH_GCOV` necessary here? It's specified in `tox.ini`.

Reply:
Oh, passenv only passes through existing variables, right? In that case I guess it's fine.

Reply:
tox.ini has ``passenv = WITH_GCOV``. It does not set ``WITH_GCOV`` but let tox pass the env vars through. By default tox unsets most env vars.